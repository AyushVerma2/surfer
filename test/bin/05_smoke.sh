#!/bin/sh

host=localhost
port=8080
url="http://${host}:${port}/"
user="one"
password="11111"

# construct curl args
args=""
args="$args -s" # NOTE turning off silent mode is useful for debugging
args="$args -m 3"
args="$args -u ${user}:${password}"

if curl $args $url ; then
    exit 0
fi

exit 1
