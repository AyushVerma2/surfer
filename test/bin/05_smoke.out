<body>
                   <h1>Welcome to surfer!!!!</h1>
                   <p><a href='/assets'>Explore imported asset list</a></p>
                   <p><a href='/api-docs'>API Documentation</a></p>
                   <p><a href='/echo'>Echo request body</a></p>
                   <p><a href='/tokens'>Manage Tokens</a></p>
                   <p><a href='/logout'>Logout</a> (click SignIn with no username/password, then Cancel)</p>
                 </body>